salvo/src/lib/benchmark/fully_dockerized_benchmark.py

```diff
@@ -0,0 +1,37 @@
+"""
+This module contains the methods to perform a nighthawk benchmark using
+containers for the scripts, nighthawk binaries, and envoy
+
+https://github.com/envoyproxy/nighthawk/blob/master/benchmarks/README.md
+"""
+
+import logging
+
+from src.lib.benchmark.base_benchmark import BaseBenchmark
+
+log = logging.getLogger(__name__)
+
+
+class Benchmark(BaseBenchmark):
+
+  def __init__(self, job_control, benchmark_name, **kwargs):
```

Comment:
Note, an initializer of a public class is also considered public and should have a docstring. When updating the other docstrings to the format recommended by the style guide, let's also add a docstring here and include an `Args:` section where we describe the arguments the initializer takes.